### ISSUE TITLE
save logs issue

### ISSUE THREAD

**Issue by sas22** [palantir/docker-compose-rule#186]
hi, i'm using version 0.28.1 version (mvn repo was not update with newer version, be happy to be notified once they are published here https://dl.bintray.com/palantir/releases/com/palantir/docker/compose/docker-compose-rule/)

There is issue when save the logs when i have more than one service in my docker-compose file. That is because it split the services array and attach the new line "\n" to it.. this result in the name of the log to break down with this new line and only the last service on that array gets to be written /collected correctly

e.g., snippet from my log shows. only the last service 'ksm' log file name is correct 'ksm.log' and not like for other servies, e.g. 'sgw\n.log'
Thanks

`2017/03/21 09:29:04.008 [FileLogCollector] [pool-5-thread-7]:   INFO:  Writing logs for container 'cdl2
' to 'C:\Users\siasaraf\workspace\drm_stage2_branch\target\docker-compose\cdl2
.log'   
2017/03/21 09:29:04.008 [FileLogCollector] [pool-5-thread-4]:   INFO:  Writing logs for container 'sgw
' to 'C:\Users\siasaraf\workspace\drm_stage2_branch\target\docker-compose\sgw
.log'   
2017/03/21 09:29:04.009 [FileLogCollector] [pool-5-thread-8]:   INFO:  Writing logs for container 'ksm' to 'C:\Users\siasaraf\workspace\drm_stage2_branch\target\docker-compose\ksm.log'   
2017/03/21 09:29:04.020 [DockerComposeRule] [main]:   DEBUG:  docker-compose cluster started  `

**Comment by joelea** [comment]
Hey @sas22 

1. There is a newer version, but the package named changed to docker-compose-rule-junit4 and a new package called docker-compose-rule-core to make space for a junit5 version that depends on core.
1. Could you post a docker-compose file that exhibits this to make sure we're fixing the right thing?

Thanks
Joe

**Comment by sas22** [comment]
yes, i saw it but perhaps its in your mvn private repo? and was not published to public one?
i cannot pull the following libraries i see you released - https://dl.bintray.com/palantir/releases/com/palantir/docker/compose/docker-compose-rule-core/

**Comment by sas22** [comment]
and thanks for the fast reply @joelea :)

**Comment by joelea** [comment]
This should be the one you want to make the rule work: https://dl.bintray.com/palantir/releases/com/palantir/docker/compose/docker-compose-rule-junit4/

Are you unable to pull that 0.31.1 of that? That seems very strange. Could you try pulling that version and dump the error output in the issue? That would be super helpful.

Thanks
Joe

**Comment by sas22** [comment]
mvn cannot find this artifact, although it could for the earlier versions before the split to the several projects. so i means it wasn't pushed to your mvn repo. where previous versions were pushed.
http://jcenter.bintray.com/com/palantir/docker/compose/

so the following pull does not work.
	<dependency>
			<groupId>com.palantir.docker.compose</groupId>
			<artifactId>docker-compose-rule-junit4</artifactId>
			<version>0.31.1</version>
		</dependency>

**Comment by joelea** [comment]
Sorry for the back and forth on this, I'm very confused as to what's going on, I've also just remembered that this issue isn't about fetching 0.31.1.

For the download issue: can you make sure you're using https://dl.bintray.com/palantir/releases as the repo as opposed to http://jcenter.bintray.com - we intend to publish to the first one and one of us needs to investigate why the jars are also ending up in the latter.

For the logs issue: could you let us know the versions of docker and docker-compose your using, which OS you're on and (if possible) a docker-compose.yml that triggers this.

Thanks for reporting the issue and sorry it's being fiddly to sort out.

**Comment by sas22** [comment]
it's ok. we will add a proxy to your new mvn repo. so you are good. you just moved probably and our proxy was not set to reflect that :+1:

**Comment by sas22** [comment]
seems the proxy didn't work. you still probably need to push it to  https://dl.bintray.com/palantir/releases
